### PR TITLE
New and refreshed I/O Redesign

### PR DESCRIPTION
**Pull request by josephgregoryii**
## Features 👨🏻‍🎨 
- New I/O Banner 
- Redesigned PackTiles
- Removal of filter display and functionality
- Improved Mobile display and functionality
- Updated result fields to display selected category

### CODE REVIEW

File: src/components/GuidedInstallTile.js [hunk 1]
```diff
@@ -74,8 +74,9 @@ const GuidedInstallTile = () => {
               }
             `}
           >
-            Install the New Relic agent with a single command line and start
-            monitoring your log and infrastructure data in real time.
+            Many engineers start here. You'll install an agent with a single
+            command line and start monitoring your log and infrastructure data
+            in real time.
```
Comment (aswanson-nr) [inline src/components/GuidedInstallTile.js:79]: Can we check with the writing team on this? "a single command line" doesn't make a lot of sense. Maybe we can change it to something like:
```suggestion
            Many engineers start here. You'll install an agent with a single
            command and start monitoring your log and infrastructure data
            in real time.
```

Comment (jpvajda) [reply src/components/GuidedInstallTile.js:79]: all the final copy is in the FIGMA file but I agree to this change. I'll note it in the FIGMA file we changed this.

**Comment by jpvajda** [comment]
The es-lint check is failing due to 

```
 /home/runner/work/developer-website/developer-website/src/components/GuidedInstallTile.js
Error:   78:74  error  Insert `·in`  prettier/prettier
Error:   79:12  error  Delete `·in`  prettier/prettier
```

**Review by aswanson-nr — APPROVED** [review]
🚀

**Pull request merged**